### DISCUSSION TITLE
Add new partition for Jetson AGX Orin devkit

### DISCUSSION THREAD

**Discussion by bagen1** [OE4T/meta-tegra#1450] [Q&A] [answered]
Hello!
I am a beginner of Yocto and I am trying to add a new partition to the rootfs, so I want to have two partitions containing the same rootfs. 
My image is based on core-image-minimal so not the tegra-demo-distro. I have looked at #858, #1245 and #1251 but I don't understand how to modify the xml file (for me the flash.xml.in I believe) or maybe I should create my own? but then I don't know where it should be placed so bitbake knows that it should use it.

best regards,
Elin

**Comment by madisongh** [comment, marked as answer]
There is some [documentation on this topic in our wiki](https://github.com/OE4T/meta-tegra/wiki/Redundant-Rootfs-A-B-Partition-Support) as well.

**Reply by bagen1** [reply]
Yes I have looked at that as well, I managed to get two partitions with the help from [this repository](https://github.com/madisongh/tegra-test-distro)
